superbuild/projects_modules/gvirtualXRay.cmake

```diff
@@ -0,0 +1,141 @@
+function(gvirtualXRay_project) 
+
+set(ep gvirtualXRay)
+
+## #############################################################################
+## List the dependencies of the project
+## #############################################################################
+
+list(APPEND ${ep}_dependencies "xcom")
+
+## #############################################################################
+## Prepare the project
+## ############################################################################# 
+
+EP_Initialisation(${ep}
+  USE_SYSTEM OFF
+  BUILD_SHARED_LIBS ON
+  REQUIRED_FOR_PLUGINS OFF
+  )
+
+if (NOT USE_SYSTEM_${ep})
+
+set(LIB_EXT "dylib")  
```

Comment:
What about Windows?

Reply:
Not yet supported. Comment the plugin in order to build for windows.

Reply:
I can add in the superbuild's CMakeLists.txt a test to not include this dependency for windows platform

Reply:
As you want, or it could wait a bit: we're in a middle of the music2->music3 phase and for now we do not code on Windows. But it would be needed for the next release indeed :)